WordPress/Classes/Services/NotificationSyncService.swift

```diff
@@ -0,0 +1,304 @@
+import Foundation
+
+
+// MARK: - Notifications
+//
+let NotificationSyncServiceDidUpdateNotifications = "NotificationSyncServiceDidUpdateNotifications"
+
+
+// MARK: - NotificationSyncService
+//
+class NotificationSyncService
+{
+    /// Returns the Main Managed Context
+    ///
+    private var contextManager = ContextManager.sharedInstance()
```

Comment:
Problem i was having, was, actually... the **dotcomAPI** getter. You cannot use *self* before initializing all of the properties. Updated as per your suggestion, thanks so much!!